client/src/PasswordReset/NewPassword.js

```diff
@@ -76,7 +77,8 @@ export const NewPassword = () => {
           error: {
             status: 401,
             attribute: 'email',
-            type: 'not_confirmed'
+            type: 'unconfirmed',
+            context: { email: data.email }
```

Comment:
This is required for the resend confirmation link rendered in the confirmation alert.